helm-charts/mock-classifier/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mock-classifier.fullname" . }}
```

Comment:
I know I requested postfixes on the other resources, but please don't add one here.